### DISCUSSION TITLE
use of NVMe

### DISCUSSION THREAD

**Discussion by zrml** [longhorn/longhorn#3554] [General]
Can we use NVMe?
What are the constraints?
Thank you very much

**Comment by PhanLe1010** [comment]
Do you mean to provide storage for Longhorn?

user provide storage for Longhorn by giving it a mount point in the filesystem. As long as the file system support spare file and direct IO, Longhorn should work. It doesn't really depend on whether the disk is connected using nvme vs sata

**Reply by zrml** [reply]
OK thanks @PhanLe1010
